### ISSUE TITLE
Multiarch "Hello world"

### ISSUE THREAD

**Issue by rhit-seilerar** [rhit-seilerar/exert-capstone#10]
Make a statically-linked "Hello World" program that can be compiled on multiple architectures.
Look into tempfs

- It'll be a C program, compiled with gcc and statically-linked to all libraries
  - Disable standard libraries, then specifically link to each?
- We'll want to compile it for i386 and ARM, and verify that it actually runs on those
  - We should be able to emulate ARM with PANDA

**Comment by rhit-morehask** [comment]
Current time spent: 2 hours (paired by Aria and Stephanie)

**Comment by rhit-morehask** [comment]
Currently stuck on copying the compiler stuff from opt/cross and into the usermode/build directory

**Comment by rhit-morehask** [comment]
Paired on by Aria and Stephanie for an additional 3 hours
Waiting on client clarification.

**Comment by rhit-seilerar** [comment]
Paired on by Aria and Stephanie for an additional hour.
Compiling seems to be working, but testing requires a frontend refactor, which we worked on.
Moving the refactor to a new task and closing this one.
